src/QirRuntime/lib/QIR/quantum__rt.hpp

```diff
@@ -91,13 +90,13 @@ extern "C"
     // ------------------------------------------------------------------------
 
     // Allocates space for a tuple requiring the given number of bytes and sets the reference count to 1.
-    QIR_SHARED_API QirTupleHeader* quantum__rt__tuple_create(int64_t); // NOLINT
+    QIR_SHARED_API char* quantum__rt__tuple_create(int64_t); // NOLINT

```

Comment:
This is for a generic memory buffer, right? Would it make sense to typedef a "buffer" type for this instead of using char* directly? Just makes it clearer it is not a string. You could even call that type QirTupleHeader, rather than buffer.

Reply:
Yes, it would. I actually have PTuple typedef exactly for this, but apparently missed it in some places.